scripts/release.js

```diff
@@ -27,6 +27,14 @@ const extensionsSourceDirectory = path.join(process.cwd(), "extensions")
 const eulaFile = path.join(process.cwd(), "Outrun-Labs-EULA-v1.1.md")
 const thirdPartyFile = path.join(process.cwd(), "ThirdPartyLicenses.txt")
```

Comment:
I just realized reading through we're missing the licenses for sparkle-osx and winsparkle in our `ThirdPartyLicenses.txt` - would you mind adding them?

Reply:
Sure!

Reply:
Thanks Zach!

Reply:
Added in ea1363a!